includes/wikia/models/DesignSystemGlobalFooterModel.class.php

```diff
@@ -28,7 +28,12 @@ public function getData() {
 		$data = [
 			'header' => [
 				'type' => 'link-image',
+				// 'image' is deprecated, use 'img' instead
 				'image' => 'wds-company-logo-fandom-powered-by-wikia-two-lines',
+				'img' => [
+					'type' => 'image-local',
+					'name' => 'wds-company-logo-fandom-powered-by-wikia-two-lines',
+				],
 				'href' => $this->getHref( 'fandom-logo' ),
 				'title' => [
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 36 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)